### ISSUE TITLE
new seed request

### ISSUE THREAD

**Issue by shemerk** [Selsion/DSPSeedScanner#4]
First of all, incredible job! well done!

I realize it's a long process, but is there a way to scan the list of 3-moon seeds and find those that also contain Fire Ice?

Thanks!

**Comment by SirWeasley** [comment]
Another one with the 3-moon seeds is that all the resources are more than 10 million each in the starting system with the fire ice.

I have been manually checking the 3-moon list.

**Comment by shemerk** [comment]
I didn't quite understand your answer...
do you mean to say it's impossible?

**Comment by SirWeasley** [comment]
no, I just have found one as manually checking each seed takes a lot of time

**Comment by shemerk** [comment]
Thanks for asnwering! 
Feel free to share the seed :)

**Comment by SirWeasley** [comment]
I found this in my search https://doubleuth.github.io/DSP-Seed-Finder/find-galaxy

**Comment by LRFalk01** [comment]
Here's an exhaustive list of seeds w/ 3 stater moons, a deuterium gas giant, and a moon with fire ice:
https://gist.github.com/LRFalk01/8cc6b8748c26d21e15c725ad14c47ed6
